packages/forms/src/text/FormikTextField.tsx

```diff
@@ -88,7 +88,9 @@ export const FormikTextField: ForwardRefExoticComponent<FormikTextFieldProps> =
           onChange={(event) => {
             onChange?.(event);
 
-            if (!event.defaultPrevented) {
+            // event.preventDefault is called on handleKeyDown for Textfield select
+            // check out SelectInput.js for more details
```

Comment:
It is better to add link to file if you want to reference something. Otherwise you can remove it. I am sure we cannot find SelectInput.js file)